app/_kong_plugins/opa/index.md

```diff
@@ -32,6 +32,137 @@ categories:
 
 search_aliases:
   - open policy agent
+
+related_resources:
+  - text: Block unauthorized requests in {{site.base_gateway}} with the OPA plugin
+    url: /how-to/block-unauthorized-requests-with-opa/
 ---
 
 ## Overview
+
+The OPA plugin allows you to forward requests to [Open Policy Agent](https://openpolicyagent.org/) and process the requests only if the authorization policy allows it.
+
+## OPA input
+
+When {{site.base_gateway}} forwards a request to OPA, the request data is sent with the following structure:
```

Comment:
```suggestion
Depending on what you configure for the plugin, the following request data is sent when {{site.base_gateway}} forwards a request to OPA:
```

Reply:
The request data sent doesn't depend on the plugin config, it depends on the request sent. And "the following request data is sent" makes it sound like the whole structure is sent, which isn't the case, it depends on the contents of the request. i'll rephrase this differently.